ReactiveCocoa/Swift/TupleExtensions.swift

```diff
@@ -16,7 +16,7 @@ internal func repack<A>(t: (), value: A) -> (A) {
 }
 
 internal func repack<A, B>(t: (A), value: B) -> (A, B) {
```

Comment:
Besides, because `repack<A>` and `repack<A, B>` are not used, should we remove them?